backend/restApi.js

```diff
@@ -22,10 +22,21 @@ router.use(jsonParser);
 
 router.post("/crossword", (req, res) => {
   const crossword = req.body;
-  if (req.body.adminToken !== process.env.ADMIN_TOKEN) {
-    res.status = 401;
-    res.send("Access denied");
-    return;
+  // If ADMIN_TOKEN is set (via command line), use it
+  if (process.env.ADMIN_TOKEN) {
+    if (req.body.adminToken !== process.env.ADMIN_TOKEN) {
+      res.status = 401;
+      res.send("Access denied");
+      return;
+    }
+  } 
+  // Otherwise ignore it and use the one defined in the .env file
+  else {
+      if (req.body.adminToken !== process.env.REACT_APP_ADMIN_TOKEN) {
```

Comment:
Separat else condition not needed, as we only care about `ADMIN_TOKEN` in this case